### PR TITLE
Clean up introductory info for EVM sidechain bridges.

### PR DESCRIPTION
**Pull request by oeggert**

### CODE REVIEW

File: docs/evm-sidechain/witness-servers.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+html: witness-server.html
```
Comment (DennisDawson) [inline docs/evm-sidechain/witness-servers.md:2]: Should be witness-servers.html

File: docs/evm-sidechain/witness-servers.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+blurb: Witness servers listen for bridge transactions and provide attestations.
+labels:
+  - Interoperability
+status: not_enabled
+---
+# Witness Servers
+
+A witness server enables bridge operators to listen for transactions on the connected chains and sign attestations. Attestations are used as proof that events on a chain occurred.
```
Comment (DennisDawson) [inline docs/evm-sidechain/witness-servers.md:10]: I might put _attestations_ in italics, since it's defined in the following sentence.

**Review by DennisDawson — APPROVED** [review]
Pretty sure you must change the reference to witness-servers.html, but I'll approve so that you can push after you make that change.

File: docs/evm-sidechain/witness-servers.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+Running a witness server requires you to:
+
+1. Create a bridge door account and be one of its witnesses or signer owners.
```
Comment (mDuo13) [inline docs/evm-sidechain/witness-servers.md:14]: Should this one have a link like the other two bullet points do?

File: docs/evm-sidechain/bridge-client.md [hunk 4]
```diff
@@ -7,6 +7,9 @@ status: not_enabled
 ---
 # Bridge Client
 
-The bridge client provides a user-friendly interfact to interact with your bridge. You can use the client to connect wallets and transfer XRP or tokens across XRPL chains.
+The bridge client provides a user-friendly interface to interact with your bridge. You can use the client to connect wallets and transfer XRP or tokens across XRPL chains.
```
Comment (mDuo13) [inline docs/evm-sidechain/bridge-client.md:10]: "friendly" is subjective, of course, and this makes as much sense without it.

```suggestion
The bridge client provides a user interface to interact with your bridge. You can use the client to connect wallets and transfer XRP or tokens across XRPL chains.
```

Comment (mDuo13) [reply docs/evm-sidechain/bridge-client.md:10]: Also, by "your" bridge, you mean this is meant to be run by the entities _running_ the bridge? Or can this be used by "common" users to interact with a bridge run by someone else?

**Review by mDuo13 — COMMENTED** [review]
Looks good although the "Witness Servers" page feels a bit stubby to me? I'm not sure which information it's missing, though.

**Pull request merged**